cg-spec/editors_draft.html

```diff
@@ -441,7 +441,24 @@ <h2>Structure of this document</h2>
     </section>
 
     <section id="conformance">
-      <div class=issue data-number="3"></div>
+      <p>A system supports RDF-star:</p>
+      <ul>
+        <li>if it supports one or more of the RDF 1.1 standard syntaxes for input, and 
+          one or more of its input syntaxes passes the tests described in the
+          <a href="https://w3c.github.io/rdf-star/tests/">RDF-star test suite</a>.
```

Comment:
Shall we be even more explicit here? See:
```suggestion
          one or more of its input syntaxes passes the tests described in the
          <a href="https://w3c.github.io/rdf-star/tests/">RDF-star test suite</a>
          (i.e., the <a href="https://w3c.github.io/rdf-star/tests/nt/syntax/">N-Triples Syntax tests</a>,
          the <a href="https://w3c.github.io/rdf-star/tests/turtle/syntax/">Turtle Syntax tests</a>, or
          the <a href="https://w3c.github.io/rdf-star/tests/trig/syntax/">TriG Syntax tests</a>).
```